File: APIGateway_with_Authorizer/APIGateway_with_CognitoUserPool/SAM/Lambda/target_lambda/target_lambda.js
exports.handler = async (event,context) => {
    console.log(JSON.stringify(event));

    return {
        statusCode: 200,
        headers : {
          "Access-Control-Allow-Origin": "*",
          "Access-Control-Allow-Methods" : "*",
          "Access-Control-Allow-Headers": "*"
        },
        body:  JSON.stringify({"message" : "OK"})
    };
};

File: APIGateway_to_ALB/CloudFormation/Lambda/target_lambda/target_lambda.py
#-*- encoding:utf-8 -*-
import json
import os
import sys
#Third Party
import boto3

def lambda_handler(event,context):
    print(json.dumps(event))
    return {
        "statusCode": 200,
        "body": json.dumps({"message" : "OK"})
   }
